data/dataverses/open-source-at-harvard/datasets/open-source-at-harvard/.filemetadata/language-count.sh

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@landreev the directory "code" be in this path to reflect the file hierarchy? The are organized like this:

```
open-source-at-harvard/files/code/language-count.sh
open-source-at-harvard/files/run.sh
open-source-at-harvard/files/results/language-count.tsv
open-source-at-harvard/files/data/2019-02-25.tsv
```

Reply:
(addressed - see below)